Makefile

```diff
@@ -57,10 +59,12 @@ CLANG ?= clang
 CLANG_VERSION = $(shell $(CLANG) --version)
 LLVM_BINDIR = $(shell $(LLVM_CONFIG) --bindir | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):/\/\1/g')
 LLVM_LIBDIR = $(shell $(LLVM_CONFIG) --libdir | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):/\/\1/g')
+# TODO: apparently no llvm_config flag to get canonical paths to tools
```

Comment:
Not sure what the todo is here. If it is at all important, add an issue before merging.